src/main/java/mate/academy/rickandmorty/controller/RickAndMortyController.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.SeriesCharacterResponseDto;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/rickandmorty")
+public class RickAndMortyController {
+    private final CharacterService characterService;
+
+    @GetMapping("/randomwiki")
+    @Operation(summary = "Get one random wiki",
+            description = "Get one random wiki about characters")
+    public SeriesCharacterResponseDto getRandomWiki() {
+        return characterService.getRandomWiki();
+    }
+
+    @GetMapping("/findbyname")
```

Comment:
imagine your site is deployed, and has domain `rickandmorty.com`. so with your endpoint, it would sound like `rickandmorty.com/rickandmorty`? a little bit odd
lets improve naming
```suggestion
@RequestMapping("/character")
public class CharacterController {
    private final CharacterService characterService;

    @GetMapping("/random")
    @Operation(summary = "Get one random character",
            description = "Get one random wiki about characters")
    public SeriesCharacterResponseDto getRandomCharacter() {
        return characterService.getRandomCharacter();
    }

    @GetMapping("/search")
```